packages/toolkit/src/view/catalog/components/CreateCatalogCard.tsx

```diff
@@ -135,46 +146,42 @@ Tokens: #: ${catalog.totalTokens || "N/A"}
                 <div className="product-headings-heading-4">{catalog.name}</div>
               </div>
               <Separator orientation="horizontal" className="my-[10px]" />
-              <p className="mb-auto line-clamp-3 product-body-text-3-regular whitespace-pre-wrap break-words">
-                {catalog.description}
-              </p>
-              {/* Coming in v2 */}
-              {/* <div className="flex flex-wrap gap-1 mt-auto">
-                {catalog.tags && catalog.tags.length > 0 ? (
-                  <>
-                    {catalog.tags.slice(0, MAX_VISIBLE_TAGS).map((tag, index) => (
-                      <Tag key={index} variant="lightBlue">
+              <ScrollArea.Root className="flex-grow overflow-hidden">
+                <p className="product-body-text-3-regular whitespace-pre-wrap break-words">
+                  {catalog.description}
+                </p>
+              </ScrollArea.Root>
+              <div className="mt-auto overflow-x-auto">
+                <div className="flex gap-1 pb-2">
+                  {catalog.tags && catalog.tags.length > 0 ? (
+                    catalog.tags.map((tag, index) => (
+                      <Tag
+                        key={index}
+                        variant="lightNeutral"
+                        className="shrink-0 !py-0.5 !px-2"
+                      >
                         {tag}
                       </Tag>
-                    ))}
-                    {catalog.tags.length > MAX_VISIBLE_TAGS && (
-                      <Button
-                        variant="tertiaryGrey"
-                        size="sm"
-                        onClick={handleTagsClick}
-                      >
-                        +{catalog.tags.length - MAX_VISIBLE_TAGS}
-                      </Button>
-                    )}
-                  </>
-                ) : (
-                  <Button
-                    variant="tertiaryGrey"
-                    size="sm"
-                    onClick={handleTagsClick}
-                  >
-                    + Tags
-                  </Button>
-                )}
-              </div> */}
+                    ))
+                  ) : (
+                    <Button
+                      variant="tertiaryGrey"
+                      size="sm"
+                      onClick={handleTagsClick}
+                    >
+                      + Tags
+                    </Button>
```

Comment:
@thewbuk This make me feel weird, so once you have tag in place, you can not add more tags?

Reply:
The only option for now is to go into edit @EiffelFly